pyproject.toml

```diff
@@ -61,10 +64,11 @@ PyQt5 = "5.14.1"
 requests = "2.27.1"
 canmatrix = "0.9.1"
 click = "7.1.2"
+atomicwrites = "^1.4.1"
```

Comment:
Why did this line get added?
```suggestion
```

Reply:
Those are temporary, added when troubleshooting the bugs on the CI run, will remove.

Reply:
Removed.